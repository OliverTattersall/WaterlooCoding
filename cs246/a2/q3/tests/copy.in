a 0 b 1 1 4 4
a 0 y 0 0 2 3
pc0
c c 0
pc0
c c 0
pc0
c m 
pc0
