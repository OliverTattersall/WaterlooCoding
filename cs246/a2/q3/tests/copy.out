Dimensions: 5x5
Number of rectangles: 2
	[Blue (1,1) 4x4]
	[Yellow (0,0) 2x3]
temporary canvas
Dimensions: 30x35
Number of rectangles: 3
	[Yellow (0,0) 2x3]
	[Yellow (20,20) 5x15]
	[Blue (0,20) 10x30]
Dimensions: 5x5
Number of rectangles: 2
	[Blue (1,1) 4x4]
	[Yellow (0,0) 2x3]
temporary canvas
Dimensions: 30x35
Number of rectangles: 3
	[Yellow (0,0) 2x3]
	[Yellow (20,20) 5x15]
	[Blue (0,20) 10x30]
Dimensions: 5x5
Number of rectangles: 2
	[Blue (1,1) 4x4]
	[Yellow (0,0) 2x3]
temporary canvas
Dimensions: 30x35
Number of rectangles: 2
	[Yellow (20,20) 5x15]
	[Blue (0,20) 10x30]
Dimensions: 5x5
Number of rectangles: 2
	[Blue (1,1) 4x4]
	[Yellow (0,0) 2x3]
